README.md

```diff
@@ -0,0 +1,42 @@
+# Metabase Node.js static embedding sample
+
+This sample code demonstrates using Metabase static embedding in Node.js using Express.
+
+# Running this sample
+
+## Pre-requisites
+* Have a running instance of Metabase. If you don't have one
+  * [download the free open source version](https://www.metabase.com/start/oss/) or
+  * [sign up for a free trial of Metabase Cloud](https://www.metabase.com/pricing/).
+* A dashboard to embed. If you don't have one, use X-Rays to let Metabase create one for you. Note down the dashboard id.
+
+## Preparing the embed
+1. Sign in to your Metabase instance as an admin. 
+2. Go to admin settings and enabling embedding.
+3. Under admin settings/embedding, click on static embedding and copy the embedding secret key.
+
+## Configure the app
+
+1. Paste the secret key into an env var: `$ export METABASE_EMBEDDING_SECRET="PASTE_SECRET_HERE"`
+2. Create an env var pointing to your Metabase site URL, if it's not on http://localhost:3000 `$ export METABASE_SITE_URL="http://localhost:4000"`
+3. Create an env var with the ID of the dashboard to embed: `$ export METABASE_EMBED_DASHBOARD_ID="8"`
+3. Install dependencies: `$ bundle install`
```

Comment:
I assume this is for Ruby? Also we already have that step in the next section on the 3rd one. 
> Install the dependencies: `$ npm install`

Maybe this step could be removed? What do you think?

Reply:
Yes, this is an oversight. And yes there is a duplicate step to install the dependencies.
I will clean it up.